renderer.php

```diff
@@ -251,7 +251,7 @@ function envbar_close(el) {
      * @param object $config Config
      * @return string Debug text
      */
-    protected function get_debug_text($canedit = false, $config) {
+    protected function get_debug_text($config, $canedit = false) {
```

Comment:
was there a reason for swapping the order of these args?

Reply:
Apparently optional params should come last in php 8+
https://www.php.net/manual/en/migration80.deprecated.php

Fixes:
Deprecated: Required parameter $config follows optional parameter $canedit in
local/envbar/renderer.php on line 254

Or would you prefer to drop the default value instead?

Reply:
Nope that makes perfect sense thanks